src/Adapter/Search/SearchProductSearchProvider.php

```diff
@@ -70,10 +73,14 @@ public function runQuery(
             );
             $products = $result['result'];
             $count    = $result['total'];
+
+            Hook::exec('actionSearch', array('expr' => $queryString, 'total' => $count));
```

Comment:
Oh, ok, sorry, so maybe we should add a second param and mark first one as a deprecated?